add `make format` to cover most auto-fixable formatting (#8022)
wagtail/wagtail: docs/contributing/python_guidelines.rst

If you have installed Wagtail's testing dependencies (``pip install -e '.[testing]'``), you can check your code by
running ``make lint``.

> You can run all Python formatting manually via ``make format``.

Reviewer: 'manually' feels like an odd word to use here... I understand it's to distinguish it from the pre-commit hook, but this command is still very much automating a manual task. I'd be inclined to just leave it out:

```suggestion
You can run all Python formatting with ``make format``.
```
